Drop AtomString's `operator NSString *()`

cdumez opened the pull request:
#### a8821c1d98c95725c59cdc9464a10c2048513d41
<pre>
Drop AtomString&apos;s `operator NSString *()`
<a href="https://bugs.webkit.org/show_bug.cgi?id=291612">https://bugs.webkit.org/show_bug.cgi?id=291612</a>

Reviewed by Geoffrey Garen.

Drop AtomString&apos;s `operator NSString *()`. Use createNSString() instead which makes it
clearer we&apos;re allocating a new object and which returns a RetainPtr instead of an
autoreleased object.

* Source/WTF/wtf/text/AtomString.h:
* Source/WebCore/editing/cocoa/HTMLConverter.mm:
(HTMLConverter::_addAttachmentForElement):
(HTMLConverter::_fillInBlock):
(HTMLConverter::_processHeadElement):
(HTMLConverter::_addLinkForElement):
(HTMLConverter::_addTableForElement):
(HTMLConverter::_processElement):
(HTMLConverter::_exitElement):
(preferredFilenameForElement):
* Source/WebCore/platform/graphics/avfoundation/objc/CDMInstanceFairPlayStreamingAVFObjC.mm:
(WebCore::CDMInstanceSessionFairPlayStreamingAVFObjC::requestLicense):
* Source/WebCore/platform/text/cocoa/LocaleCocoa.mm:
(WebCore::LocaleCocoa::LocaleCocoa):
(WebCore::LocaleCocoa::canonicalLanguageIdentifierFromString):
* Source/WebKit/UIProcess/mac/WebDateTimePickerMac.mm:
(-[WKDateTimePicker updatePicker:]):
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKDOMElement.mm:
(-[WKDOMElement getAttribute:]):
* Source/WebKitLegacy/mac/DOM/DOMCustomXPathNSResolver.mm:
(DOMCustomXPathNSResolver::lookupNamespaceURI):
* Source/WebKitLegacy/mac/DOM/DOMEvent.mm:
(-[DOMEvent type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLAnchorElement.mm:
(-[DOMHTMLAnchorElement charset]):
(-[DOMHTMLAnchorElement coords]):
(-[DOMHTMLAnchorElement download]):
(-[DOMHTMLAnchorElement hreflang]):
(-[DOMHTMLAnchorElement name]):
(-[DOMHTMLAnchorElement ping]):
(-[DOMHTMLAnchorElement rel]):
(-[DOMHTMLAnchorElement rev]):
(-[DOMHTMLAnchorElement shape]):
(-[DOMHTMLAnchorElement target]):
(-[DOMHTMLAnchorElement type]):
(-[DOMHTMLAnchorElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLAreaElement.mm:
(-[DOMHTMLAreaElement alt]):
(-[DOMHTMLAreaElement coords]):
(-[DOMHTMLAreaElement ping]):
(-[DOMHTMLAreaElement rel]):
(-[DOMHTMLAreaElement shape]):
(-[DOMHTMLAreaElement target]):
(-[DOMHTMLAreaElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBRElement.mm:
(-[DOMHTMLBRElement clear]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBaseElement.mm:
(-[DOMHTMLBaseElement target]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBaseFontElement.mm:
(-[DOMHTMLBaseFontElement color]):
(-[DOMHTMLBaseFontElement face]):
(-[DOMHTMLBaseFontElement size]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBodyElement.mm:
(-[DOMHTMLBodyElement aLink]):
(-[DOMHTMLBodyElement background]):
(-[DOMHTMLBodyElement bgColor]):
(-[DOMHTMLBodyElement link]):
(-[DOMHTMLBodyElement text]):
(-[DOMHTMLBodyElement vLink]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLButtonElement.mm:
(-[DOMHTMLButtonElement type]):
(-[DOMHTMLButtonElement name]):
(-[DOMHTMLButtonElement value]):
(-[DOMHTMLButtonElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLDivElement.mm:
(-[DOMHTMLDivElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLDocument.mm:
(-[DOMHTMLDocument dir]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLElement.mm:
(-[DOMHTMLElement title]):
(-[DOMHTMLElement lang]):
(-[DOMHTMLElement dir]):
(-[DOMHTMLElement webkitdropzone]):
(-[DOMHTMLElement accessKey]):
(-[DOMHTMLElement idName]):
(-[DOMHTMLElement autocapitalize]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLEmbedElement.mm:
(-[DOMHTMLEmbedElement align]):
(-[DOMHTMLEmbedElement name]):
(-[DOMHTMLEmbedElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFontElement.mm:
(-[DOMHTMLFontElement color]):
(-[DOMHTMLFontElement face]):
(-[DOMHTMLFontElement size]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFormElement.mm:
(-[DOMHTMLFormElement acceptCharset]):
(-[DOMHTMLFormElement autocomplete]):
(-[DOMHTMLFormElement name]):
(-[DOMHTMLFormElement target]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFrameElement.mm:
(-[DOMHTMLFrameElement frameBorder]):
(-[DOMHTMLFrameElement longDesc]):
(-[DOMHTMLFrameElement marginHeight]):
(-[DOMHTMLFrameElement marginWidth]):
(-[DOMHTMLFrameElement name]):
(-[DOMHTMLFrameElement scrolling]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFrameSetElement.mm:
(-[DOMHTMLFrameSetElement cols]):
(-[DOMHTMLFrameSetElement rows]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHRElement.mm:
(-[DOMHTMLHRElement align]):
(-[DOMHTMLHRElement size]):
(-[DOMHTMLHRElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHeadingElement.mm:
(-[DOMHTMLHeadingElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHtmlElement.mm:
(-[DOMHTMLHtmlElement version]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLIFrameElement.mm:
(-[DOMHTMLIFrameElement align]):
(-[DOMHTMLIFrameElement frameBorder]):
(-[DOMHTMLIFrameElement height]):
(-[DOMHTMLIFrameElement longDesc]):
(-[DOMHTMLIFrameElement marginHeight]):
(-[DOMHTMLIFrameElement marginWidth]):
(-[DOMHTMLIFrameElement name]):
(-[DOMHTMLIFrameElement sandbox]):
(-[DOMHTMLIFrameElement scrolling]):
(-[DOMHTMLIFrameElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLImageElement.mm:
(-[DOMHTMLImageElement name]):
(-[DOMHTMLImageElement align]):
(-[DOMHTMLImageElement alt]):
(-[DOMHTMLImageElement srcset]):
(-[DOMHTMLImageElement sizes]):
(-[DOMHTMLImageElement useMap]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLInputElement.mm:
(-[DOMHTMLInputElement accept]):
(-[DOMHTMLInputElement alt]):
(-[DOMHTMLInputElement dirName]):
(-[DOMHTMLInputElement formTarget]):
(-[DOMHTMLInputElement max]):
(-[DOMHTMLInputElement min]):
(-[DOMHTMLInputElement name]):
(-[DOMHTMLInputElement pattern]):
(-[DOMHTMLInputElement placeholder]):
(-[DOMHTMLInputElement step]):
(-[DOMHTMLInputElement type]):
(-[DOMHTMLInputElement defaultValue]):
(-[DOMHTMLInputElement selectionDirection]):
(-[DOMHTMLInputElement align]):
(-[DOMHTMLInputElement useMap]):
(-[DOMHTMLInputElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLIElement.mm:
(-[DOMHTMLLIElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLabelElement.mm:
(-[DOMHTMLLabelElement htmlFor]):
(-[DOMHTMLLabelElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLegendElement.mm:
(-[DOMHTMLLegendElement align]):
(-[DOMHTMLLegendElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLinkElement.mm:
(-[DOMHTMLLinkElement charset]):
(-[DOMHTMLLinkElement hreflang]):
(-[DOMHTMLLinkElement media]):
(-[DOMHTMLLinkElement rel]):
(-[DOMHTMLLinkElement rev]):
(-[DOMHTMLLinkElement target]):
(-[DOMHTMLLinkElement type]):
(-[DOMHTMLLinkElement as]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMapElement.mm:
(-[DOMHTMLMapElement name]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMarqueeElement.mm:
(-[DOMHTMLMarqueeElement behavior]):
(-[DOMHTMLMarqueeElement bgColor]):
(-[DOMHTMLMarqueeElement direction]):
(-[DOMHTMLMarqueeElement height]):
(-[DOMHTMLMarqueeElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMediaElement.mm:
(-[DOMHTMLMediaElement mediaGroup]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMetaElement.mm:
(-[DOMHTMLMetaElement content]):
(-[DOMHTMLMetaElement httpEquiv]):
(-[DOMHTMLMetaElement name]):
(-[DOMHTMLMetaElement scheme]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLModElement.mm:
(-[DOMHTMLModElement dateTime]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLOListElement.mm:
(-[DOMHTMLOListElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLObjectElement.mm:
(-[DOMHTMLObjectElement code]):
(-[DOMHTMLObjectElement align]):
(-[DOMHTMLObjectElement archive]):
(-[DOMHTMLObjectElement border]):
(-[DOMHTMLObjectElement codeBase]):
(-[DOMHTMLObjectElement codeType]):
(-[DOMHTMLObjectElement height]):
(-[DOMHTMLObjectElement name]):
(-[DOMHTMLObjectElement standby]):
(-[DOMHTMLObjectElement type]):
(-[DOMHTMLObjectElement useMap]):
(-[DOMHTMLObjectElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLOptGroupElement.mm:
(-[DOMHTMLOptGroupElement label]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLParagraphElement.mm:
(-[DOMHTMLParagraphElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLParamElement.mm:
(-[DOMHTMLParamElement name]):
(-[DOMHTMLParamElement type]):
(-[DOMHTMLParamElement value]):
(-[DOMHTMLParamElement valueType]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLScriptElement.mm:
(-[DOMHTMLScriptElement htmlFor]):
(-[DOMHTMLScriptElement event]):
(-[DOMHTMLScriptElement charset]):
(-[DOMHTMLScriptElement type]):
(-[DOMHTMLScriptElement nonce]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLSelectElement.mm:
(-[DOMHTMLSelectElement name]):
(-[DOMHTMLSelectElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLStyleElement.mm:
(-[DOMHTMLStyleElement media]):
(-[DOMHTMLStyleElement type]):
(-[DOMHTMLStyleElement nonce]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableCaptionElement.mm:
(-[DOMHTMLTableCaptionElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableCellElement.mm:
(-[DOMHTMLTableCellElement align]):
(-[DOMHTMLTableCellElement axis]):
(-[DOMHTMLTableCellElement bgColor]):
(-[DOMHTMLTableCellElement ch]):
(-[DOMHTMLTableCellElement chOff]):
(-[DOMHTMLTableCellElement headers]):
(-[DOMHTMLTableCellElement height]):
(-[DOMHTMLTableCellElement vAlign]):
(-[DOMHTMLTableCellElement width]):
(-[DOMHTMLTableCellElement abbr]):
(-[DOMHTMLTableCellElement scope]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableColElement.mm:
(-[DOMHTMLTableColElement align]):
(-[DOMHTMLTableColElement ch]):
(-[DOMHTMLTableColElement chOff]):
(-[DOMHTMLTableColElement vAlign]):
(-[DOMHTMLTableColElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableElement.mm:
(-[DOMHTMLTableElement align]):
(-[DOMHTMLTableElement bgColor]):
(-[DOMHTMLTableElement border]):
(-[DOMHTMLTableElement cellPadding]):
(-[DOMHTMLTableElement cellSpacing]):
(-[DOMHTMLTableElement frameBorders]):
(-[DOMHTMLTableElement rules]):
(-[DOMHTMLTableElement summary]):
(-[DOMHTMLTableElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableRowElement.mm:
(-[DOMHTMLTableRowElement align]):
(-[DOMHTMLTableRowElement bgColor]):
(-[DOMHTMLTableRowElement ch]):
(-[DOMHTMLTableRowElement chOff]):
(-[DOMHTMLTableRowElement vAlign]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableSectionElement.mm:
(-[DOMHTMLTableSectionElement align]):
(-[DOMHTMLTableSectionElement ch]):
(-[DOMHTMLTableSectionElement chOff]):
(-[DOMHTMLTableSectionElement vAlign]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTextAreaElement.mm:
(-[DOMHTMLTextAreaElement dirName]):
(-[DOMHTMLTextAreaElement name]):
(-[DOMHTMLTextAreaElement placeholder]):
(-[DOMHTMLTextAreaElement wrap]):
(-[DOMHTMLTextAreaElement type]):
(-[DOMHTMLTextAreaElement selectionDirection]):
(-[DOMHTMLTextAreaElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLUListElement.mm:
(-[DOMHTMLUListElement type]):
* Source/WebKitLegacy/mac/DOM/DOMKeyboardEvent.mm:
(-[DOMKeyboardEvent keyIdentifier]):
* Source/WebKitLegacy/mac/DOM/DOMNode.mm:
(-[DOMNode namespaceURI]):
(-[DOMNode prefix]):
(-[DOMNode localName]):
(-[DOMNode lookupPrefix:]):
(-[DOMNode lookupNamespaceURI:]):
* Source/WebKitLegacy/mac/DOM/DOMTokenList.mm:
(-[DOMTokenList value]):
(-[DOMTokenList item:]):
* Source/WebKitLegacy/mac/DOM/DOMXPath.mm:
(-[DOMNativeXPathNSResolver lookupNamespaceURI:]):
* Source/WebKitLegacy/mac/History/WebHistoryItem.mm:
(-[WebHistoryItem description]):
* Source/WebKitLegacy/mac/Plugins/WebBasePluginPackage.mm:
(-[WebBasePluginPackage MIMETypeForExtension:]):
* Source/WebKitLegacy/mac/Plugins/WebPluginDatabase.mm:
(-[WebPluginDatabase refresh]):
(-[WebPluginDatabase _removePlugin:]):
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
(-[WebFrame name]):

Canonical link: <a href="https://commits.webkit.org/293773@main">https://commits.webkit.org/293773@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f6b3e1c36c82041f90fefd3140ca365aa8ac223e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/99826 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/19471 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/9749 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/104951 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/50404 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/101867 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/19776 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/27907 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/75990 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/33075 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/102833 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/15072 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/90141 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/56355 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/14878 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/8128 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/49776 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/92482 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/84797 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/8213 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/107312 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/98432 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/26937 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/19669 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/84939 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/27298 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/86346 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/84466 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/21464 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/29142 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/6857 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/20742 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/26873 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/32084 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/122058 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/26684 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/34079 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/30001 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/28245 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WebKitLegacy/mac/History/WebHistoryItem.mm:
```diff
@@ -230,8 +230,8 @@ - (NSString *)description
     if (!coreItem->target().isEmpty()) {
-        NSString *target = coreItem->target();
-        [result appendFormat:@" in \"%@\"", target];
+        RetainPtr target = coreItem->target().createNSString();
+        [result appendFormat:@" in \"%@\"", target.get()];
```
```suggestion
        [result:appendFormar:@" in\"%@\"", core item->target().createNSString().get()];
```

cdumez replied:
Following up in https://github.com/WebKit/WebKit/pull/44177

cdumez committed:
Drop AtomString's `operator NSString *()`
https://bugs.webkit.org/show_bug.cgi?id=291612

Reviewed by Geoffrey Garen.

Drop AtomString's `operator NSString *()`. Use createNSString() instead which makes it
clearer we're allocating a new object and which returns a RetainPtr instead of an
autoreleased object.

* Source/WTF/wtf/text/AtomString.h:
* Source/WebCore/editing/cocoa/HTMLConverter.mm:
(HTMLConverter::_addAttachmentForElement):
(HTMLConverter::_fillInBlock):
(HTMLConverter::_processHeadElement):
(HTMLConverter::_addLinkForElement):
(HTMLConverter::_addTableForElement):
(HTMLConverter::_processElement):
(HTMLConverter::_exitElement):
(preferredFilenameForElement):
* Source/WebCore/platform/graphics/avfoundation/objc/CDMInstanceFairPlayStreamingAVFObjC.mm:
(WebCore::CDMInstanceSessionFairPlayStreamingAVFObjC::requestLicense):
* Source/WebCore/platform/text/cocoa/LocaleCocoa.mm:
(WebCore::LocaleCocoa::LocaleCocoa):
(WebCore::LocaleCocoa::canonicalLanguageIdentifierFromString):
* Source/WebKit/UIProcess/mac/WebDateTimePickerMac.mm:
(-[WKDateTimePicker updatePicker:]):
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKDOMElement.mm:
(-[WKDOMElement getAttribute:]):
* Source/WebKitLegacy/mac/DOM/DOMCustomXPathNSResolver.mm:
(DOMCustomXPathNSResolver::lookupNamespaceURI):
* Source/WebKitLegacy/mac/DOM/DOMEvent.mm:
(-[DOMEvent type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLAnchorElement.mm:
(-[DOMHTMLAnchorElement charset]):
(-[DOMHTMLAnchorElement coords]):
(-[DOMHTMLAnchorElement download]):
(-[DOMHTMLAnchorElement hreflang]):
(-[DOMHTMLAnchorElement name]):
(-[DOMHTMLAnchorElement ping]):
(-[DOMHTMLAnchorElement rel]):
(-[DOMHTMLAnchorElement rev]):
(-[DOMHTMLAnchorElement shape]):
(-[DOMHTMLAnchorElement target]):
(-[DOMHTMLAnchorElement type]):
(-[DOMHTMLAnchorElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLAreaElement.mm:
(-[DOMHTMLAreaElement alt]):
(-[DOMHTMLAreaElement coords]):
(-[DOMHTMLAreaElement ping]):
(-[DOMHTMLAreaElement rel]):
(-[DOMHTMLAreaElement shape]):
(-[DOMHTMLAreaElement target]):
(-[DOMHTMLAreaElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBRElement.mm:
(-[DOMHTMLBRElement clear]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBaseElement.mm:
(-[DOMHTMLBaseElement target]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBaseFontElement.mm:
(-[DOMHTMLBaseFontElement color]):
(-[DOMHTMLBaseFontElement face]):
(-[DOMHTMLBaseFontElement size]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLBodyElement.mm:
(-[DOMHTMLBodyElement aLink]):
(-[DOMHTMLBodyElement background]):
(-[DOMHTMLBodyElement bgColor]):
(-[DOMHTMLBodyElement link]):
(-[DOMHTMLBodyElement text]):
(-[DOMHTMLBodyElement vLink]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLButtonElement.mm:
(-[DOMHTMLButtonElement type]):
(-[DOMHTMLButtonElement name]):
(-[DOMHTMLButtonElement value]):
(-[DOMHTMLButtonElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLDivElement.mm:
(-[DOMHTMLDivElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLDocument.mm:
(-[DOMHTMLDocument dir]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLElement.mm:
(-[DOMHTMLElement title]):
(-[DOMHTMLElement lang]):
(-[DOMHTMLElement dir]):
(-[DOMHTMLElement webkitdropzone]):
(-[DOMHTMLElement accessKey]):
(-[DOMHTMLElement idName]):
(-[DOMHTMLElement autocapitalize]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLEmbedElement.mm:
(-[DOMHTMLEmbedElement align]):
(-[DOMHTMLEmbedElement name]):
(-[DOMHTMLEmbedElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFontElement.mm:
(-[DOMHTMLFontElement color]):
(-[DOMHTMLFontElement face]):
(-[DOMHTMLFontElement size]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFormElement.mm:
(-[DOMHTMLFormElement acceptCharset]):
(-[DOMHTMLFormElement autocomplete]):
(-[DOMHTMLFormElement name]):
(-[DOMHTMLFormElement target]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFrameElement.mm:
(-[DOMHTMLFrameElement frameBorder]):
(-[DOMHTMLFrameElement longDesc]):
(-[DOMHTMLFrameElement marginHeight]):
(-[DOMHTMLFrameElement marginWidth]):
(-[DOMHTMLFrameElement name]):
(-[DOMHTMLFrameElement scrolling]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLFrameSetElement.mm:
(-[DOMHTMLFrameSetElement cols]):
(-[DOMHTMLFrameSetElement rows]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHRElement.mm:
(-[DOMHTMLHRElement align]):
(-[DOMHTMLHRElement size]):
(-[DOMHTMLHRElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHeadingElement.mm:
(-[DOMHTMLHeadingElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLHtmlElement.mm:
(-[DOMHTMLHtmlElement version]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLIFrameElement.mm:
(-[DOMHTMLIFrameElement align]):
(-[DOMHTMLIFrameElement frameBorder]):
(-[DOMHTMLIFrameElement height]):
(-[DOMHTMLIFrameElement longDesc]):
(-[DOMHTMLIFrameElement marginHeight]):
(-[DOMHTMLIFrameElement marginWidth]):
(-[DOMHTMLIFrameElement name]):
(-[DOMHTMLIFrameElement sandbox]):
(-[DOMHTMLIFrameElement scrolling]):
(-[DOMHTMLIFrameElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLImageElement.mm:
(-[DOMHTMLImageElement name]):
(-[DOMHTMLImageElement align]):
(-[DOMHTMLImageElement alt]):
(-[DOMHTMLImageElement srcset]):
(-[DOMHTMLImageElement sizes]):
(-[DOMHTMLImageElement useMap]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLInputElement.mm:
(-[DOMHTMLInputElement accept]):
(-[DOMHTMLInputElement alt]):
(-[DOMHTMLInputElement dirName]):
(-[DOMHTMLInputElement formTarget]):
(-[DOMHTMLInputElement max]):
(-[DOMHTMLInputElement min]):
(-[DOMHTMLInputElement name]):
(-[DOMHTMLInputElement pattern]):
(-[DOMHTMLInputElement placeholder]):
(-[DOMHTMLInputElement step]):
(-[DOMHTMLInputElement type]):
(-[DOMHTMLInputElement defaultValue]):
(-[DOMHTMLInputElement selectionDirection]):
(-[DOMHTMLInputElement align]):
(-[DOMHTMLInputElement useMap]):
(-[DOMHTMLInputElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLIElement.mm:
(-[DOMHTMLLIElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLabelElement.mm:
(-[DOMHTMLLabelElement htmlFor]):
(-[DOMHTMLLabelElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLegendElement.mm:
(-[DOMHTMLLegendElement align]):
(-[DOMHTMLLegendElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLLinkElement.mm:
(-[DOMHTMLLinkElement charset]):
(-[DOMHTMLLinkElement hreflang]):
(-[DOMHTMLLinkElement media]):
(-[DOMHTMLLinkElement rel]):
(-[DOMHTMLLinkElement rev]):
(-[DOMHTMLLinkElement target]):
(-[DOMHTMLLinkElement type]):
(-[DOMHTMLLinkElement as]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMapElement.mm:
(-[DOMHTMLMapElement name]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMarqueeElement.mm:
(-[DOMHTMLMarqueeElement behavior]):
(-[DOMHTMLMarqueeElement bgColor]):
(-[DOMHTMLMarqueeElement direction]):
(-[DOMHTMLMarqueeElement height]):
(-[DOMHTMLMarqueeElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMediaElement.mm:
(-[DOMHTMLMediaElement mediaGroup]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLMetaElement.mm:
(-[DOMHTMLMetaElement content]):
(-[DOMHTMLMetaElement httpEquiv]):
(-[DOMHTMLMetaElement name]):
(-[DOMHTMLMetaElement scheme]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLModElement.mm:
(-[DOMHTMLModElement dateTime]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLOListElement.mm:
(-[DOMHTMLOListElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLObjectElement.mm:
(-[DOMHTMLObjectElement code]):
(-[DOMHTMLObjectElement align]):
(-[DOMHTMLObjectElement archive]):
(-[DOMHTMLObjectElement border]):
(-[DOMHTMLObjectElement codeBase]):
(-[DOMHTMLObjectElement codeType]):
(-[DOMHTMLObjectElement height]):
(-[DOMHTMLObjectElement name]):
(-[DOMHTMLObjectElement standby]):
(-[DOMHTMLObjectElement type]):
(-[DOMHTMLObjectElement useMap]):
(-[DOMHTMLObjectElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLOptGroupElement.mm:
(-[DOMHTMLOptGroupElement label]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLParagraphElement.mm:
(-[DOMHTMLParagraphElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLParamElement.mm:
(-[DOMHTMLParamElement name]):
(-[DOMHTMLParamElement type]):
(-[DOMHTMLParamElement value]):
(-[DOMHTMLParamElement valueType]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLScriptElement.mm:
(-[DOMHTMLScriptElement htmlFor]):
(-[DOMHTMLScriptElement event]):
(-[DOMHTMLScriptElement charset]):
(-[DOMHTMLScriptElement type]):
(-[DOMHTMLScriptElement nonce]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLSelectElement.mm:
(-[DOMHTMLSelectElement name]):
(-[DOMHTMLSelectElement type]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLStyleElement.mm:
(-[DOMHTMLStyleElement media]):
(-[DOMHTMLStyleElement type]):
(-[DOMHTMLStyleElement nonce]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableCaptionElement.mm:
(-[DOMHTMLTableCaptionElement align]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableCellElement.mm:
(-[DOMHTMLTableCellElement align]):
(-[DOMHTMLTableCellElement axis]):
(-[DOMHTMLTableCellElement bgColor]):
(-[DOMHTMLTableCellElement ch]):
(-[DOMHTMLTableCellElement chOff]):
(-[DOMHTMLTableCellElement headers]):
(-[DOMHTMLTableCellElement height]):
(-[DOMHTMLTableCellElement vAlign]):
(-[DOMHTMLTableCellElement width]):
(-[DOMHTMLTableCellElement abbr]):
(-[DOMHTMLTableCellElement scope]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableColElement.mm:
(-[DOMHTMLTableColElement align]):
(-[DOMHTMLTableColElement ch]):
(-[DOMHTMLTableColElement chOff]):
(-[DOMHTMLTableColElement vAlign]):
(-[DOMHTMLTableColElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableElement.mm:
(-[DOMHTMLTableElement align]):
(-[DOMHTMLTableElement bgColor]):
(-[DOMHTMLTableElement border]):
(-[DOMHTMLTableElement cellPadding]):
(-[DOMHTMLTableElement cellSpacing]):
(-[DOMHTMLTableElement frameBorders]):
(-[DOMHTMLTableElement rules]):
(-[DOMHTMLTableElement summary]):
(-[DOMHTMLTableElement width]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableRowElement.mm:
(-[DOMHTMLTableRowElement align]):
(-[DOMHTMLTableRowElement bgColor]):
(-[DOMHTMLTableRowElement ch]):
(-[DOMHTMLTableRowElement chOff]):
(-[DOMHTMLTableRowElement vAlign]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTableSectionElement.mm:
(-[DOMHTMLTableSectionElement align]):
(-[DOMHTMLTableSectionElement ch]):
(-[DOMHTMLTableSectionElement chOff]):
(-[DOMHTMLTableSectionElement vAlign]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLTextAreaElement.mm:
(-[DOMHTMLTextAreaElement dirName]):
(-[DOMHTMLTextAreaElement name]):
(-[DOMHTMLTextAreaElement placeholder]):
(-[DOMHTMLTextAreaElement wrap]):
(-[DOMHTMLTextAreaElement type]):
(-[DOMHTMLTextAreaElement selectionDirection]):
(-[DOMHTMLTextAreaElement accessKey]):
* Source/WebKitLegacy/mac/DOM/DOMHTMLUListElement.mm:
(-[DOMHTMLUListElement type]):
* Source/WebKitLegacy/mac/DOM/DOMKeyboardEvent.mm:
(-[DOMKeyboardEvent keyIdentifier]):
* Source/WebKitLegacy/mac/DOM/DOMNode.mm:
(-[DOMNode namespaceURI]):
(-[DOMNode prefix]):
(-[DOMNode localName]):
(-[DOMNode lookupPrefix:]):
(-[DOMNode lookupNamespaceURI:]):
* Source/WebKitLegacy/mac/DOM/DOMTokenList.mm:
(-[DOMTokenList value]):
(-[DOMTokenList item:]):
* Source/WebKitLegacy/mac/DOM/DOMXPath.mm:
(-[DOMNativeXPathNSResolver lookupNamespaceURI:]):
* Source/WebKitLegacy/mac/History/WebHistoryItem.mm:
(-[WebHistoryItem description]):
* Source/WebKitLegacy/mac/Plugins/WebBasePluginPackage.mm:
(-[WebBasePluginPackage MIMETypeForExtension:]):
* Source/WebKitLegacy/mac/Plugins/WebPluginDatabase.mm:
(-[WebPluginDatabase refresh]):
(-[WebPluginDatabase _removePlugin:]):
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
(-[WebFrame name]):

Canonical link: https://commits.webkit.org/293773@main
